lib/runner.js

```diff
@@ -529,7 +533,13 @@ Runner.prototype.runTests = function (suite, fn) {
     }
 
     if (test.isPending()) {
-      self.emit('pending', test);
+      if (self.forbidPending) {
+        test.isPending = alwaysFalse;
```

Comment:
I'm not entirely happy with this hack to circumvent `Runner.prototype.fail`'s check that ignores pending tests based on `test.isPending()`, but I'm not sure if there's a better option. Any opinions would be appreciated, provided we can avoid stalling over the matter.